diff --git a/node_modules/react-native/local-cli/runIOS/findMatchingSimulator.js b/node_modules/react-native/local-cli/runIOS/findMatchingSimulator.js
index 494ab08..321eadb 100644
--- a/node_modules/react-native/local-cli/runIOS/findMatchingSimulator.js
+++ b/node_modules/react-native/local-cli/runIOS/findMatchingSimulator.js
@@ -27,9 +27,9 @@ function findMatchingSimulator(simulators, simulatorName) {
   var match;
   for (let version in devices) {
     // Making sure the version of the simulator is an iOS (Removes Apple Watch, etc)
-    if (version.indexOf('iOS') !== 0) {
-      continue;
-    }
+    // if (version.indexOf('iOS') !== 0) {
+    //   continue;
+    // }
     for (let i in devices[version]) {
       let simulator = devices[version][i];
       // Skipping non-available simulator
